frontend/packages/erd-core/vitest.config.ts

```diff
@@ -1,11 +1,15 @@
 // biome-ignore lint/correctness/noNodejsModules: Because this file is a config file
 import * as path from 'node:path'
+import react from '@vitejs/plugin-react'
 import { defineConfig } from 'vitest/config'
 
 export default defineConfig({
+  plugins: react(),
```

Comment:
Reference for setup `@vitejs/plugin-react`: https://github.com/vitejs/vite-plugin-react/tree/6db7e7c95826009c7db20b990cdc874569967489/packages/plugin-react